packages/inventory/src/components/table/EntityTableToolbar.js

```diff
@@ -92,7 +93,10 @@ const EntityTableToolbar = ({
      */
     const debounceGetAllTags = useCallback(debounce((config, options) => {
         if (showTags && !hasItems) {
-            dispatch(fetchAllTags(config, options));
+            dispatch(fetchAllTags({
+                ...customFilters,
+                config
```

Comment:
@karelhala here's a question. Do we want the config to have a higher priority? Also, I think the `...config` should be spread as well.

Reply:
Good point with the spread! And yes, we want the config to have higher priority.